index.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+
+<html lang="en">
+
+<head>
+
+    <meta charset="UTF-8">
+
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+
+    <title>Document</title>
+
+    <link  rel="stylesheet" href="styles.css">
+
+</head>
+
+<body>
+
+    <header>
+
+ 
+
+    </header>
```

Comment:
The head tag is empty if you are not putting anything inside it I kindly suggest that you remove it, so as to make the code look clean.

Reply:
Good job with this review 🔥.